packages/twenty-server/src/engine/api/graphql/workspace-query-builder/factories/foreign-data-wrapper-query.factory.ts

```diff
@@ -25,7 +25,7 @@ export class ForeignDataWrapperQueryFactory {
     foreignDataWrapperId: string,
     userMappingOptions: UserMappingOptions,
   ) {
-    return `CREATE USER MAPPING IF NOT EXISTS FOR ${userMappingOptions.username} SERVER "${foreignDataWrapperId}" OPTIONS (user '${userMappingOptions.username}', password '${userMappingOptions.password}')`;
+    return `CREATE USER MAPPING IF NOT EXISTS FOR CURRENT_USER SERVER "${foreignDataWrapperId}" OPTIONS (user '${userMappingOptions.username}', password '${userMappingOptions.password}')`;
```

Comment:
CURRENT_USER @thomtrp I'm wondering if this works as intended. CURRENT_USER returns the user of the current context but since we are sharing the same datasource for every tenant it might not be the one you expect?

Reply:
As discussed this should work as it is but it won't work when we will have a different user per workspace, please add a comment to detail this 🙏